packages/core/src/editor/store.tsx

```diff
@@ -101,6 +101,7 @@ export const useEditorStore = (
     ActionMethodsWithConfig,
     {
       ...editorInitialState,
+      ...(options.handlers ? { handlers: options.handlers } : {}),
```

Comment:
I'm pretty sure this doesn't do anything since the `handlers` property is no longer part of the state